assets/src/hooks/useKeyDown.tsx

```diff
@@ -1,18 +1,18 @@
 import { useEffect } from 'react';
 
 export function useKeyDown(
-  callback: () => void,
+  callback: (ctrlKey?: boolean, metaKey?: boolean, shiftKey?: boolean) => any,
   keyCodes: string[],
   options: { ctrlKey?: boolean } = {},
   dependencies: any = [],
 ): void {
-  const handler = ({ metaKey, ctrlKey, code }: KeyboardEvent) => {
+  const handler = ({ metaKey, ctrlKey, shiftKey, code }: KeyboardEvent) => {
     const { ctrlKey: ctrlRequired } = options;
 
     const ctrlMatch =
       ctrlRequired === undefined || ctrlKey === ctrlRequired || metaKey === ctrlRequired;
     if (ctrlMatch && keyCodes.includes(code)) {
-      callback();
+      callback(ctrlKey, metaKey, shiftKey);
```

Comment:
Instead of writing code for handling custom use cases for windows and Mac, I am passing these parameters to the callback functions so that the customer handling can be done there.